### DISCUSSION TITLE
How to restore system settings after uninstall AlDente

### DISCUSSION THREAD

**Discussion by menduo** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1180] [Q&A] [answered]
Hello, thank you for developing this AlDente, it's great. However, for some reasons, I recently uninstalled it. 

I have a question to consult. After installing the AlDente Pro, it prompted me to turn off some system settings, but now I have forgotten which settings they are. Now that I have uninstalled them, I need to restore these system settings. 

Where can I find this setting list?

Thank you!

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @menduo,

The only system setting we ask our users to deactivate is the "Optimized battery charging".
You can reactivate it in macOS settings.

Best,
Matthias

**Reply by menduo** [reply]
Thank you.
